serveradmin/servershell/static/js/servershell.js

```diff
@@ -208,21 +208,21 @@ servershell.alert = function(text, level, auto_dismiss=5) {
         'dark'
     ];
 
-    if (!levels.includes(level))
+    if (!levels.includes(level)) {
         return;
+    }
 
-    let box = $('#js-alert');
-    levels.forEach(level => box.removeClass(`alert-${level}`));
-    box.addClass(`alert-${level}`);
-
-    let message = $('#js-alert-message');
-    message.text(text);
+    let template = $('.alert').first();
 
-    box.toggle();
+    let new_alert = template.clone();
+    new_alert.addClass(`alert-${level}`);
+    new_alert.children('.alert-text').text(text);
+    template.after(new_alert);
```

Comment:
Let's group all the alerts in an extra `div` with a class/id which you can reuse here and append on that group instead of putting it after the template. I would suggest to separate the template too, so maybe:
``` html
<div.alert-group#alert-group>
    <div.alert-template#alert-template />
    <div.alerts#alerts>
        <div.alert />
    </div>
</div>
```
Then you can find the template with `$('#alert-template')` and add new alerts (without knowing the exact method right now) with `$('#alerts').append(alert)`.

Reply:
Why is that better ?

Reply:
Clarified in a call. That makes because the position of the template should not matter and this way we can append the new alert within the box. I will adjust the code.